### PR TITLE
Judge steganographic payload dimension

### LINKED ISSUE #436: Judge image payload dimension before send into decoding
Image payload should meets the dimension `width: 1024px` and `height: 1240px` requirement before decoding it.

### PR DESCRIPTION
**Pull request by guanbinrui**
This PR includes:

+ get image dimension from binary;
+ judge image payload dimension before decoding it;

close #436

### CODE REVIEW

**Comment by guanbinrui** [comment]
It's not urgent to merge this PR. Maybe after we merged release/1.8.0.

File: src/utils/image.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        case 'image/jpeg':
+            return getDimensionAsJPEG(buf) ?? fallback
+        case 'image/png':
+            return getDimensionAsPNG(buf)
+        default:
+            return fallback
+    }
+}
+
+function getDimensionAsPNG(buf: Uint8Array) {
```
Comment (Jack-Works) [inline src/utils/image.ts]: it will be better to have a comment that refer to the spec of PNG and JPEG

Comment (neruthes) [reply src/utils/image.ts]: A link to https://github.com/DimensionDev/Maskbook/issues/304 can reduce bytes used.

Comment (Jack-Works) [reply src/utils/image.ts]: > A link to #304 can reduce bytes used.

I mean the spec of PNG, not our Maskbook rfc

**Pull request merged**